RUFAS/biophysical/animal/animal_module_constants.py

```
@@ -219,3 +239,339 @@ class AnimalModuleConstants:
+        "Piedmontese": 1.2,
+        "Pinzgauer": 1.2,
+        "Polled Hereford": 1.2,
+        "Red Poll": 1.2,
+        "Sahiwal": 1.2,
+        "Salers": 1.2,
+        "Santa Gertrudis": 1.2,
+        "Shorthorn": 1.2,
+        "Simmental": 1.0,
+        "South Devon": 1.2,
+        "Tarentaise": 1.2,
+    }
```

Ruff (0.15.17)

[warning] Mutable default value for class attribute

(RUF012)